docs/Docs.md

```diff
@@ -130,11 +130,10 @@ protocol period `T`시간동안 `M_i`이 `M_j`에게 `ping`을 보내는데 성
 
 * {Suspect Ml, inc=i} overrides
     - {Suspect Ml, inc=j}, i>j
-    - {Alive Ml, inc=j}, ij
+    - {Alive Ml, inc=j}, i>j
 
```

Comment:
Did you say that `{Alive Ml, inc=j}, i>j` => `{Alive Ml, inc=j}, i>=j`

Reply:
good point! 👍